src/PhenX.EntityFrameworkCore.BulkInsert.SqlServer/SqlServerBulkInsertProvider.cs

```diff
@@ -22,7 +22,7 @@ public SqlServerBulkInsertProvider(ILogger<SqlServerBulkInsertProvider>? logger
 
     //language=sql
     /// <inheritdoc />
-    protected override string AddTableCopyBulkInsertId => $"ALTER TABLE {{0}} ADD {BulkInsertId} INT IDENTITY PRIMARY KEY;";
+    protected override string AddTableCopyBulkInsertId => $"ALTER TABLE {{0}} ADD {BulkInsertId} INT;";
```

Comment:
I added this column as identity primary for future features like identity output, is it a problem for MySQL ? I'm not sure making it an option is a responsibility we want to expose to the user

Reply:
I have tested something and have forgotten to revert it. Good catch.